snap/snapcraft.yaml

```diff
@@ -108,6 +107,8 @@ parts:
     - -DLLVM_BINUTILS_INCDIR=/usr/include
     - -DLLVM_TARGETS_TO_BUILD=AArch64;ARM;Mips;MSP430;NVPTX;PowerPC;X86
     - -DLLVM_EXPERIMENTAL_TARGETS_TO_BUILD=RISCV;WebAssembly
+    - -DLLVM_ENABLE_TERMINFO=OFF
+    - -DLLVM_ENABLE_LIBEDIT=OFF
```

Comment:
The LLVM 8 fork adds `SPIRV` support.  Is that something we can/should add an extra target for?

Reply:
It's a separate extra library, src included as git submodule, and built automatically, no conventional LLVM target.